src/modelbench/benchmark_runner.py

```diff
@@ -322,15 +252,23 @@ def __init__(self, test_run: TestRunBase):
 
     def handle_item(self, item: TestRunItem):
         for sut in self.test_run.suts:
-            self.downstream_put(TestRunItem(item.test, item.test_item, sut))
+            run_item = TestRunItem(item.test, item.test_item, sut)
+            self.test_run.journal.item_entry(
+                "queuing item",
+                run_item,
+                prompt=item.prompt_with_context().prompt.text,
+            )
+
+            self.downstream_put(run_item)
 
 
 class TestRunSutWorker(IntermediateCachingPipe):
     def __init__(self, test_run: TestRunBase, cache: MBCache, thread_count=1):
+
         super().__init__(cache, thread_count)
         self.test_run = test_run
 
-    def handle_item(self, item):
+    def handle_item(self, item: TestRunItem):
         mg_sut = item.sut.instance(self.test_run.secrets)
         raw_request = mg_sut.translate_text_prompt(item.prompt_with_context().prompt)
```

Comment:
Can we record the sut-specific request? To 1) ensure that the SUT options were translated as expected and 2) keep a record of the actual request objects.

Reply:
Sure, no problem.